lib/secure_headers.rb

```diff
@@ -163,7 +163,7 @@ def opt_out_of_all_protection(request)
     # returned is meant to be merged into the header value from `@app.call(env)`
     # in Rack middleware.
     def header_hash_for(request)
-      config = config_for(request, prevent_dup = true)
+      config = config_for(request, _prevent_dup = true)
```

Comment:
I agree, kwargs would be a logical change here ✅  

To get around the warning, this would probably work too:

```ruby
prevent_dup = true
config = config_for(request, prevent_dup)
```